packages/layers/pay/tests/e2e/components/ConnectFeeWidget/default.spec.ts

```diff
@@ -11,18 +11,20 @@ test.describe('Connect Fee (default)', () => {
     await page.goto('./examples/components/ConnectFeeWidget/default')
     await page.waitForURL('*/**/examples/components/ConnectFeeWidget/default')
     await expect(page.getByRole('heading', { name: 'Connect Fee Widget (Basic Example)' })).toBeVisible()
+    // toggle fees to show non-loaded state
```

Comment:
So the purpose of this test is to verify the state of things before the fee is toggled on to verify the placeholder state. It should already be awaiting the initFee api call before entering this test so does this need to change (maybe it needs a half second for the init function to finish / update the html)?
It might be worth reworking this so that it verifies the loading state in test 1, clicking a button to initialize in test 2, and then doing toggle fee in test 3 since these are all the different base states of the component